documentation/slate/source/includes/stuff.md

```diff
@@ -26,6 +26,13 @@ Journey qualification process
 
 Since Navitia can return several journeys, it tags them to help the user
 choose the best one that matches their needs.
+Here are some tagging rules:
+-    there is only one "Best" itinerary. It is the ASAP one, on best time
```

Comment:
I don't understand the end of the sentence. You mean "It's the ASAP one, and if 2 are arriving at the same time the shortest one" ?